src/include/parser/explain_statement.h

```diff
@@ -0,0 +1,36 @@
+//===----------------------------------------------------------------------===//
+//
+//                         Peloton
+//
+// explain_statement.h
+//
+// Identification: src/include/parser/explain_statement.h
+//
+// Copyright (c) 2015-2018, Carnegie Mellon University Database Group
+//
+//===----------------------------------------------------------------------===//
+
+#pragma once
+
+#include "parser/sql_statement.h"
+
+namespace peloton {
+
+namespace parser {
+
+/**
+ * @class ExplainStatement
```

Comment:
Remove class name from comment.

Reply:
There are many files where this style is being followed, i.e., include the class name in the comment. See update_statement.h or insert_statement.h, etc. I just followed the same pattern here. I think, if we want to change these kind of things, it makes more sense to do it separately for all of the classes in a separate PR.

Reply:
Huh, didn't realize this has proliferated in parser code. Sure, leave it the way it is.